cli/enable/enable.go

```diff
@@ -0,0 +1,66 @@
+package enable
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// defaultDirPermissions is rights used to create folders.
+// 0755 - drwxr-xr-x
+// We need to give permission for all to execute
+// read,write for user and only read for others.
+const defaultDirPermissions = 0755
+
+// Enable creates a symbolic link in 'instances_enabled' directory
+// to a script or an application directory.
+func Enable(path string, cliOpts *config.CliOpts) error {
+	var err error
+
+	if _, err := os.Stat(cliOpts.Env.InstancesEnabled); os.IsNotExist(err) {
+		err = os.MkdirAll(cliOpts.Env.InstancesEnabled, defaultDirPermissions)
+		if err != nil {
+			return fmt.Errorf("unable to create %s\n Error: %s",
+				cliOpts.Env.InstancesEnabled, err)
+		}
+		log.Infof("created instances_enabled directory: %s", cliOpts.Env.InstancesEnabled)
+	}
+
+	if !util.DirIsWritable(cliOpts.Env.InstancesEnabled) {
```

Comment:
We can skip this check. Check for writable dir is required for `tt install` because git clone + build are heavy operations. So, it is better to check dir permissions in advance, than fail with error after 5 mins of building.
It is ok here to just return error from symlink creation, which will be 'permission denied'. Anyway it's up to you.